NativeApp/src/components/User/container.js

```diff
@@ -1,27 +1,30 @@
 import React, { Component } from 'react';
-import { getUserHolidays } from '../../services/userService';
+import { getTakenHolidays, getRemainingHolidays } from '../../utilities/holidays';
 
 export default Container => class extends Component {
   constructor(props) {
     super(props);
     this.state = {
       id: '',
       takenHolidays: [],
-      remainingHolidays: [],
+      remainingHolidays: 0,
     };
   }
 
   componentDidMount() {
-    getUserHolidays()
+    getTakenHolidays()
       .then(data => this.setState({ takenHolidays: data }));
+
+    getRemainingHolidays()
+      .then(data => this.setState({ remainingHolidays: data }));
```

Comment:
generally a good practice to stay away from ambiguously named variables like `data`, I appreciate you've copied the code from above but this might be nicer as something like...
`.then(remainingHolidays => this.setState({ remainingHolidays }));`
could even change the code above 👀

Reply:
Totally agree with you. Also makes more sense and easier to understand what data your actually passing in.